frontend/src/components/phones/onboarding/RelayNumberPicker.tsx

```diff
@@ -103,35 +95,26 @@ const RelayNumberSelection = (props: RelayNumberSelectionProps) => {
   const [relayNumberIndex, setRelayNumberIndex] = useState(0);
 
   useEffect(() => {
```

Comment:
Now that the side effect (the GET request) is contained in a hook, there's no longer a need to do this in `useEffect` anymore.

Reply:
Cleaned up